applications/serial_lte_modem/src/slm_at_httpc.c

```diff
@@ -0,0 +1,736 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include <logging/log.h>
+#include <zephyr.h>
+#include <stdio.h>
+#include <net/socket.h>
+#include <net/tls_credentials.h>
+#include <net/http_client.h>
+#include <nrf_socket.h>
+#include "slm_at_httpc.h"
+#include "slm_util.h"
+
+LOG_MODULE_REGISTER(httpc, CONFIG_SLM_LOG_LEVEL);
+
+#define HTTPC_METHOD_LEN	20
+#define HTTPC_HOST_LEN		64
+#define HTTPC_RES_LEN		256
+#define HTTPC_HEADER_LEN	512
+#define HTTPC_BUF_LEN           NET_IPV4_MTU
+#define HTTPC_REQ_TO		10
```

Comment:
I'd add unit here, for clarity `HTTPC_REQ_TO_S`.

Reply:
Fixed.